src/main/java/seedu/dietbook/checker/InputChecker.java

```diff
@@ -247,9 +247,9 @@ public static void checkGender(String userInput) throws DietException {
      */
     public static void checkAgeLimit(int age) throws DietException {
         if (age < 0) {
-            throw new DietException("Input value cannot be less than 0!");
+            throw new DietException("Age value cannot be less than 0!");
         } else if (age > AGE_CAP) {
-            throw new DietException("Input value cannot be more than 125!");
+            throw new DietException("Age value cannot be more than 125!");
```

Comment:
Perhaps consider changing 125 to AGE_CAP or 150. You might like to consider this suggestion for the other methods as well.